site/es/blog/new-in-devtools-92/index.md

```diff
@@ -0,0 +1,224 @@
+
+Veamos un ejemplo. La redeclaración de `const` se admite en scripts REPL separados (mira el ejemplo de la variable `a` más abajo). Ten en cuenta que lo siguiente no se soporta por diseño:
+
+- `const` declaradas en scripts de la página no se pueden redeclarar en scripts REPL.
+- `const` no se puede redeclarar en el mismo script REPL (mira el ejemplo de la variable `b`).
+
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/tJCPlokvxw6OWyCAmocM.png", alt="Redeclaraciones de constantes", width="800", height="496" %}
+
+{# https://chromium.googlesource.com/v8/v8/+/0acdf36510e72d5dac5777d893e77716235b7c39 #}
+
+Chromium issue: [1076427](https://crbug.com/1076427)
+
+## Visor de orden de los elementos {: #source-order }
+
+Ahora puedes ver el orden de los elementos de la página en tu pantalla para mejorar la inspección de la accesibilidad.
+
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/2QoBtjGjFxgDAkKaO3y2.png", alt="Visor de orden del código", width="800", height="515" %}
+
+El orden del contenido en un un documento HTML es muy importante para el SEO y la accesibildad. Las nuevas funcionalidades de CSS permite a los desarrolladores crear contenido que parezca muy diferente en el orden en pantalla que como está plasmado en el documento HTML. Este es un problema de accesibilidad muy grande ya que un usuario que usa un lector de pantallas puede tener una diferente, y seguramente confusa, experiencia que un usuario que usa un navegador.
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/7f83e4b0190ed2dbc32feef6d8b0315279ad7d07 #}
+
+Chromium issue: [1094406](https://crbug.com/1094406)
+
+## Nuevos accesos directos a los detalles de un iframe {: #frame-details }
+
+Ahora puedes ver los detalles de un `iframe` al hacer clic con el botón secundario en un elemento `iframe` y seleccionar la opción **Show frame details**.
+
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/YdENg6wjsgPNyMODdOHC.png", alt="Show frame details", width="800", height="486" %}
+
+Esto te llevará a una nueva vista con todos los detalles en el panel de `Application`, donde puedes examinar todos los detalles del documento, como seguridad, estado de isolación, política de permisos y poder depurar posibles problemas.
+
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/hEsg9Mc95n7w2tPrv6KH.png", alt="Detalles de la vista Frame", width="800", height="516" %}
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/30ba780ff36307473aee2e2c959287ea8d0b3bd7 #}
+
+Chromium issue: [1192084](https://crbug.com/1192084)
+
+## Mejor soporte para depurar CORS {: #cors }
+
+Los errores de *Cross-origin resource sharing (CORS)* ahora aparecen en la pestaña de *Issues* y, además, muestra las diferencias razones por las que se producen los errores. Puedes hacer click en cada *Issue* para ver el detalle de la causa y las soluciones.
```

Comment:
The details shown for each issue are suggestions and thus I consider important to keep the 'potential' adjective:
```suggestion
Los errores de *Cross-origin resource sharing (CORS)* ahora aparecen en la pestaña de *Issues* y, además, se muestran las diferentes razones por las que se producen los errores. Puedes hacer click en cada *Issue* para ver el detalle de las potenciales causas y las soluciones.
```